scripts/macos/hide_mac_app.sh

```diff
@@ -0,0 +1,8 @@
+defaults write /Library/LaunchAgents/com.jumpcloud.jcagent-tray Disabled -bool true
+mkdir -p /opt/jc_user_ro
+touch /opt/jc_user_ro/dont_show_tray_app
+PERMISSIONS="$(ls -l /opt/jc_user_ro/dont_show_tray_app | cut -c -10)"
+if [ "$PERMISSIONS" != "-rw-r--r--" ]; then
+        echo "Permissions not right for file:  $PERMISSIONS"
```

Comment:
indentations should be two spaces, we use the google shell style guide as a rule https://google.github.io/styleguide/shell.xml.